en/access-dashboard.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Access TiDB Dashboard
+summary: Learn how to access TiDB Dashboard in Kubernetes.
+category: how-to
+---
+
+# Access TiDB Dashboard
+
+TiDB Dashboard is a visualized tool in TiDB 4.0 used to monitor and diagnose the TiDB cluster. For details, see [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard).
+
+This document describes how to access TiDB Dashboard in Kubernetes.
+
+## Quick start
+
+> **Note:**
+>
+> This guide shows how to quickly access TiDB Dashboard. Do not use this method in production.
+
+TiDB Dashboard is built-in the PD component in 4.0 versions. You can refer to the following example to quickly deploy a 4.0.0-rc TiDB cluster in Kubernetes.
+
+1. Deploy the following `.yaml` file into the Kubernetes cluster by running the `kubectl apply -f` command:
+
+    ```yaml
+    apiVersion: pingcap.com/v1alpha1
+    kind: TidbCluster
+    metadata:
+    name: basic
+    spec:
+    version: v4.0.0-rc
+    timezone: UTC
+    pvReclaimPolicy: Delete
+    pd:
+        baseImage: pingcap/pd
+        replicas: 1
+        requests:
+        storage: "1Gi"
+        config: {}
+    tikv:
+        baseImage: pingcap/tikv
+        replicas: 1
+        requests:
+        storage: "1Gi"
+        config: {}
+    tidb:
+        baseImage: pingcap/tidb
+        replicas: 1
+        service:
+        type: ClusterIP
+        config: {}
+    ```
+
+2. After the cluster is created, expose TiDB to the local machine by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl port-forward svc/tidb-pd -n ${namespace} 2379:2379
+    ```
+
+3. Visit <http://localhost:2379/dashboard> in your browser to access TiDB Dashboard.
+
+## Access TiDB Dashboard by Ingress
+
+> **Note:**
+>
+> It is recommended to expose the TiDB Dashboard service using Ingress in critical production environments.
+>
+> Because the built-in TiDB Dashboard uses the same port as PD APIs, if you expose TiDB Dashboard in other methods, note that the interfaces related to PD APIs should not be exposed.
```

Comment:
```suggestion
> Because the built-in TiDB Dashboard uses the same port as PD APIs do, if you expose TiDB Dashboard in other methods, note that the interfaces related to PD APIs should not be exposed.
```